app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -28,13 +28,19 @@ def prov_field_changed
       @record =
         if @edit[:new][:src_configured_system_ids].present?
           PhysicalServer.where(:id => @edit[:new][:src_configured_system_ids].first).first
-        elsif @edit[:new][:src_vm_id].first.present?
-          MiqTemplate.where(:id => @edit[:new][:src_vm_id].first).first
+        else
+          MiqTemplate.where(:id => @edit[:new][:src_vm_id].kind_of?(Array) ? @edit[:new][:src_vm_id].first : @edit[:new][:src_vm_id]).first
```

Comment:
`MiqTemplate.where(:id => Array(@edit[:new][:src_vm_id]).first)`

and my eyes hurt less